ecs-cli/modules/cli/servicediscovery/servicediscovery_helper.go

```diff
@@ -226,36 +222,53 @@ func getSDSCFNParams(namespaceID, sdsName, networkMode string, input *utils.Serv
 	cfnParams.Add(parameterKeyNamespaceID, namespaceID)
 	cfnParams.Add(parameterKeySDSName, sdsName)
 
-	dnsType := getDNSType(input, networkMode)
+	if description := input.ServiceDiscoveryService.Description; description != "" {
+		cfnParams.Add(parameterKeySDSDescription, description)
+	}
+
+	dnsType := getDNSType(input.ServiceDiscoveryService, networkMode, true)
```

Comment:
why do we want to warn re: record type here, but not on line 108?

Reply:
I wanted the "Defaulting DNS Type to..." message to be printed only once, but wanted to re-use the code in this function in 2 different places.

Reply:
Oh, so will they always both be executed on the same action?

Reply:
Yeah, they will. The usage on line 108 is during the validation step, and the usage on 229 sets the value on the final input object.